### PR TITLE
Update language Italian.0.9.0part2.js

### PR DESCRIPTION
**Pull request by francescaneri**
edited translation. added words. Please check @jasoncalabrese @MilosKozak if you can fit change words in English in the menu to make it easier if I change before melting. thank you.

### CODE REVIEW

File: lib/language.js [hunk 1]
```diff
@@ -5026,6 +5045,7 @@ function init() {
       ,fi: 'Kertoo insuliinin tyypillisen vaikutusajan. Vaihtelee potilaan ja insuliinin tyypin mukaan. Tyypillisesti 3-4 tuntia pumpuissa käytettävällä insuliinilla.'
       ,bg: 'Представя типичната продължителност на действието на инсулина. Варира между отделните пациенти и различни инсулини. Обикновено е 3-4 часа за пациентите с помпа. Нарича се още живот на инсулина '
       ,pl: 'Odzwierciedla czas dzialania insuliny.Może różnić sie w zależnosci od chorego i rodzaju insuliny. Zwykle sa to 3-4 godziny dla insuliny podawanej pompa u wiekszosci chorych. Inna nazwa to czas trwania insuliny.'
+      ,it: 'Rappresenta la tipica duranta di funzionamento dell\'insulina. E 'diverso in ogni paziente e per ogni tipologia d\'insulina. Tipicamente è di 3-4 ore.'
       }
     ,'Insulin to carb ratio (I:C)' : {
```
Comment (jasoncalabrese) [inline lib/language.js]: Codacy Issue found: [Missing semicolon.](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/1574569969/issues/source?bid=2452379&fileBranchId=3022887#l5050)

File: lib/language.js [hunk 2]
```diff
@@ -5026,6 +5045,7 @@ function init() {
       ,fi: 'Kertoo insuliinin tyypillisen vaikutusajan. Vaihtelee potilaan ja insuliinin tyypin mukaan. Tyypillisesti 3-4 tuntia pumpuissa käytettävällä insuliinilla.'
       ,bg: 'Представя типичната продължителност на действието на инсулина. Варира между отделните пациенти и различни инсулини. Обикновено е 3-4 часа за пациентите с помпа. Нарича се още живот на инсулина '
       ,pl: 'Odzwierciedla czas dzialania insuliny.Może różnić sie w zależnosci od chorego i rodzaju insuliny. Zwykle sa to 3-4 godziny dla insuliny podawanej pompa u wiekszosci chorych. Inna nazwa to czas trwania insuliny.'
+      ,it: 'Rappresenta la tipica duranta di funzionamento dell\'insulina. E 'diverso in ogni paziente e per ogni tipologia d\'insulina. Tipicamente è di 3-4 ore.'
```
Comment (jasoncalabrese) [inline lib/language.js]: Codacy Issue found: [Missing semicolon.](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/1574569969/issues/source?bid=2452379&fileBranchId=3022887#l5048)

File: lib/language.js (hunk 1 above)
Comment (jasoncalabrese) [inline lib/language.js]: Codacy Issue found: [Expected an assignment or function call and instead saw an expression.](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/1574569969/issues/source?bid=2452379&fileBranchId=3022887#l5050)

File: lib/language.js (hunk 2 above)
Comment (jasoncalabrese) [inline lib/language.js]: Codacy Issue found: [Expected an assignment or function call and instead saw an expression.](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/1574569969/issues/source?bid=2452379&fileBranchId=3022887#l5048)

File: lib/language.js [hunk 3]
```diff
@@ -5261,12 +5304,29 @@ function init() {
...
+      ,ro: 'Start senzor'
+      ,bg: 'Стартиране на сензор'
+      ,hr: 'Start senzora'
+      ,it: 'SAGE-Avvio sensore'
+      ,dk: 'Sensorstart'
+      ,fi: 'Sensorin aloitus'
+      ,nb: 'Sensorstart'
+      ,he: 'אתחול חיישן סוכר'
+      ,pl: 'Start nowego sensora'
   };
```
Comment (jasoncalabrese) [inline lib/language.js]: Codacy Issue found: [Missing semicolon.](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/1574793244/issues/source?bid=2452379&fileBranchId=3022887#l5330)

**Comment by francescaneri** [comment]
HI, @jasoncalabrese, the lines 3468, line 3485 are fine line as content? Change the other

**Comment by MilosKozak** [comment]
Look here https://github.com/nightscout/cgm-remote-monitor/pull/1411/files
Pls do not change keys like `IOB-Insulin-on-Board` or add duplicates like 'SAGE Sensor Age' at the end.
When you go through the diff above you'll see you touched many other languages. That must be fixed.
In your PR must be only lines for your language
,it: '.....'

**Comment by francescaneri** [comment]
@MilosKozak  You have to do cleaning code and create the right keys as CGM sensor start and SAGE and plug sensor age. You try to call them the same way, and delete old ones that are not used type start DexCom sensor. tell me what you think. Matteo

**Comment by MilosKozak** [comment]
yeah. there are some keys not changed yet. you can propose change (fix it) in separate PR if you find such a bug. Only doing such changes together with translations makes it hard to review and verify.
here include only `,it: '.....'` and it can be immediately accepted and merged
thanks for understanding

**Pull request closed without merging**